wrapper/src/main/java/software/amazon/jdbc/targetdriverdialect/GenericTargetDriverDialect.java

```diff
@@ -34,6 +37,70 @@ public class GenericTargetDriverDialect implements TargetDriverDialect {
 
   private static final Logger LOGGER =
       Logger.getLogger(GenericTargetDriverDialect.class.getName());
+  public static final String CONN_GET_AUTO_COMMIT = "Connection.getAutoCommit";
```

Comment:
Can you follow the pattern used for `NETWORK_BOUND_METHODS`? Instead of listing all of the methods one by one they are stored directly in a list `List<String> NETWORK_BOUND_METHODS`.

Reply:
This list should only contain the common methods that are required by all target driver dialects

Reply:
Added this. There aren't too many methods that are common between all of them though